docker-compose.yml

```diff
@@ -7,18 +8,33 @@ redis:
   image: redis
 elasticsearch:
   image: elasticsearch
-web:
-  build: .
+
+# Web server, routing /api and / to server, client respectively
+nginx:
+  build: ./scripts
+  dockerfile: Dockerfile.nginx
+  links: [client, server]
+  ports: [80]
+
+# Rails server
+server: &rails
+  links: [postgres, redis, elasticsearch]
+  build: ./server
   volumes:
-    - ./:/opt/hummingbird/
-  ports:
-    - 3000:3000
-    - 4200
-    - 49152:49152
-  links:
-    - postgres
-    - redis
-    - elasticsearch
+    - ./server:/opt/hummingbird/server
   environment:
     RAILS_ENV: development
 
+# Sidekiq
+sidekiq:
+  <<: *rails
+  command: [sidekiq]
+
+# Ember server
+client:
+  build: ./client
+  volumes:
+    - ./client:/opt/hummingbird/client
+    - /opt/hummingbird/client/node_modules
```

Comment:
It's always node-sass which just randomly fails to build.  It's completely random as far as I can tell, since just running `docker build .` over and over will eventually get it to succeed even though the lower layers don't rebuild